internal/cypress/config.go

```diff
@@ -208,8 +208,8 @@ func loadCypressConfiguration(rootDir string, cypressCfgFile, sauceIgnoreFile st
 		return Config{}, err
 	}
 
-	if cfg.FixturesFolder != "" {
-		if err = checkAvailability(filepath.Join(configDir, cfg.FixturesFolder), true); err != nil {
+	if f, ok := cfg.FixturesFolder.(string); ok && f != "" {
```

Comment:
Could you please add a reference to https://docs.cypress.io/guides/references/configuration#Folders-Files here, so that it's a bit clearer later on why you are making a string cast here 🙏

Reply:
sure!